src/configuration.ts

```diff
@@ -277,6 +277,12 @@ export function getRelativePath(directory?: string | null, relativeTo?: string)
     return undefined;
 }
 
+// check if directory should be used as path or if it should be resolved like a module
+export function useAsPath(directory: string) {
+    return directory.startsWith("/") || directory.startsWith("./") || directory.startsWith("../")
```

Comment:
this should probably include paths starting with `node_modules/`

to simplify the check, you can use this regex:
```ts
/^(?:\.?\.?(?:\/|$)|node_modules\/)/.test(directory)
```